en/config/README.md

```diff
@@ -1,53 +1,77 @@
 # Standard Configuration
 
-This section describes the standard software configuration and calibration that is required for all PX4 vehicles.
+This section covers the common software configuration and calibration needed for most PX4 vehicles, irrespective of their type or particular configuration.
 
-:::note
-Before starting this section you should [Download QGroundControl](http://qgroundcontrol.com/downloads/) and install it on your **desktop** computer (*QGroundControl* does not support vehicle configuration on mobile platforms).
+:::tip
+For vehicle-specific configuration topics see [multicopters](../config_mc/README.md), [helicopter](../config_heli/README.md), [planes](../config_fw/README.md), or [VTOL](../config_vtol/README.md).
+:::
+
+## Preconditions
 
-To access **Vehicle Setup** screens, open the QGC application menu ("Q" icon in the top-left corner) and then choose **Vehicle Setup** in the *Select Tool* popup:
+Before starting this section you should [Download QGroundControl](http://qgroundcontrol.com/downloads/) and install it on your **desktop** computer.
+Then open the QGC application menu ("Q" icon in the top-left corner) and choose **Vehicle Setup** in the _Select Tool_ popup:
 
 ![QGC Main Menu Popup: highlighting Vehicle Setup](../../assets/qgc/setup/menu_setup.png)
-:::
 
-The sub topics cover each of the steps in detail (first install the PX4 firmware and set your vehicle airframe):

```

Comment:
Are there any bits of this config you would do if you got an RTF vehicle from a manufacturer? My assumption is "maybe compass" if you got a warning about it. The rest should be setup in an RTF correct right?

Everything else you would probably do once when bringing up your vehicle first time. If you changed something that affected weight you'd then do the autotune again, but the rest probably doesn't change right?

Reply:
It really depends. RTF could be everything from
- a PX4-based product you don't have to configure anything except follow the wizard
all the way to
- some product that was made compatible and you flash and set everything up yourself possibly even with some workarounds for specifics

> If you changed something that affected weight you'd then do the autotune again

I'd generally expect this to end in a worse result than keeping a good manufacturer tuning even if the weight changed. What you might want to adjust is the hover thrust but even that is learned automatically for climb rate-controlled modes.

Reply:
THanks for clarifying - a good reminder to me that I need to be precise. I meant [fully assembled PX4 installed](http://docs.px4.io/main/en/complete_vehicles/#fully-assembled-px4-preinstalled).

@MaEtUgR So what I think you are saying is that for such a vehicle, even if you modify it then probably you should stick with manufacturer configuration - there is no need for tuning. W.r.t. compass I've been told that generally a reasonable compass part should not "drift" but it can do so under some circumstances - so this is a "maybe" if you get warnings.

I'd expect the [UAV Kits](http://docs.px4.io/main/en/complete_vehicles/#uav-kits) with PX4 might also fit into this category?

If however you have the RTF which can run PX4 but has not been set up for PX4 you might be starting from scratch. I guess if QGC provides a particular airframe config you could rely on it being able to fly out of the box.

Reply:
> even if you modify it then probably you should stick with manufacturer configuration - there is no need for tuning

Yes, if there is a manufacturer tuning for this exact geometry and drive train then it's usually better to not start retuning except if you see issues that are clearly tuning-related. By tuning I mean controller gains and limits and IMU filtering.

> W.r.t. compass I've been told that generally a reasonable compass part should not "drift" but it can do so under some circumstances

Yeah, the compass is susceptible to many factors e.g. magnetization of parts, rewiring, environment, ... We have by now quite some mechanisms to try and make the best out of it (will be the topic of Mathieu's dev summit talk) but it's never wrong to calibrate the compass and verify against the map the absolute heading being stable in all cardinal directions.

> I guess if QGC provides a particular airframe config you could rely on it being able to fly out of the box.

We have to be very specific. In the past most cases weren't so clear. e.g. the F450 airframe, was it tuned with the original drive train? does a user build it with those components? Many people including me just used the frame which only gives size information but doesn't allow a good out-of-the-box tuning.